src/packages/emmett-sqlite/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@event-driven-io/emmett-sqlite",
+  "version": "0.22.1",
+  "type": "module",
```

Comment:
**FIX:** @davecosec, could you add `"private": true` setting (see e.g. https://github.com/event-driven-io/emmett/blob/d3303a42568fa418f96bf5fafa40a2f6851fc648/src/packages/emmett-tests/package.json#L4) then we can control when this package will be released.